doc/guides/weight_normalization.rst

```diff
@@ -0,0 +1,19 @@
+Weight normalization
+====================
+
+Let's say that you have a neuron with incoming synaptic weights :math:`w_1, w_2, \ldots, w_n`. A plasticity rule might require that the vector norm :math:`|w|` remains constant.
+
+Because of the way that the data structures are arranged in NEST, normalizing the weights per neuron is a costly operation (in terms of time spent). One has to iterate over all the neurons, then for each neuron fetch all of its connections, calculate the vector norm and perform the actual normalisation, and finally to write back the new weights.
+
+This would look something like:
+
+.. code-block:: python
+
+   for neur in neurons_to_be_normalised:
+       connect = nest.GetConnections(target=[neur], synapse_model="stdp_synapse")
+       w = nest.GetStatus(connect, "weight")
```

Comment:
Wouldn't it be more NEST3 conform to use `.get()` und `.set()`?

Reply:
isn't there even a `connect.weight` property?

Reply:
According to the transition guide, https://nest-simulator.readthedocs.io/en/latest/guides/nest2_to_nest3/nest2_to_nest3_detailed_transition_guide.html, nest.GetConnections() is still the suggested approach.